ci/lint/06_script.sh

```diff
@@ -6,13 +6,13 @@
 
 export LC_ALL=C
 
+GIT_HEAD=$(git rev-parse HEAD)
 if [ -n "$CIRRUS_PR" ]; then
-  # CIRRUS_PR will be present in a Cirrus environment. For builds triggered
-  # by a pull request this is the name of the branch targeted by the pull request.
-  # https://cirrus-ci.org/guide/writing-tasks/#environment-variables
-  COMMIT_RANGE="$CIRRUS_BRANCH..HEAD"
+  git fetch $CIRRUS_REPO_CLONE_URL $CIRRUS_BASE_SHA
```

Comment:
Is this needed?

Reply:
Actually, it replaces the `git fetch` command from the removed `ci/lint/05_before_script.sh`

Oh, I see. The `CIRRUS_BASE_SHA` must already be in the commit history for pulls, right?
I'll push another change without this command.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/20697#issuecomment-748869426).